devito/dle/parallelizer.py

```diff
@@ -190,8 +187,9 @@ def __init__(self, key=None):
             self.key = key
         else:
             self.key = lambda i: i.is_ParallelRelaxed and not i.is_Vectorizable
-        self.nthreads = NThreads()
-        self.nthreads_nested = NThreadsNested()
+        self.nthreads = NThreads(aliases='nthreads0')
+        self.nthreads_nested = NThreadsNested(aliases='nthreads1')
```

Comment:
?

Reply:
what's the question ?

Reply:
But why the aliases and not just keep the default names?